src/client/components/carousel.js

```diff
@@ -171,7 +171,7 @@ export class Carousel extends Component {
             abstractDiv,
             h('div.carousel-doc-footer', [
               h('div.carousel-doc-text', doc.text),
-              h('div.carousel-doc-datestamp', formatDistanceToNow( new Date( doc.lastEditedDate || 0 ), { addSuffix: true } ))
```

Comment:
Shouldn't that data exist for all allowable statuses?  If it doesn't exist, then the data ought to be amended.

Reply:
Maybe initialize the document 'lastEditedDate' to `createdDate`?

Reply:
That would be a reasonable fallback